src/Symfony/Component/Runtime/Runner/Middleware/MiddlewareInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Runtime\Runner\Middleware;
+
+/**
+ * @author Sascha Heilmeier <sascha.heilmeier@netlogix.de>
+ */
+interface MiddlewareInterface
+{
+    public function wrap(callable $handler, array $server): void;
```

Comment:
precise types of arguments should be documented in phpdoc.

Reply:
I tried to describe it